app/src/main/kotlin/io/eugenethedev/taigamobile/ui/screens/commontask/CommonTaskScreen.kt

```diff
@@ -141,10 +144,13 @@ fun CommonTaskScreen(
         toolbarSubtitle = projectName,
         commonTask = commonTask.data,
         creator = creator.data,
+        isLoading = commonTask is LoadingResult,
```

Comment:
Why though? I usually try to place parameters in the same order as they apper in the function declaration. But I don't see a problem if some of them placed differently since these are named arguments thus it won't significantly affect readability